src/main/java/kr/codesqaud/cafe/dto/comment/CommentListDto.java

```diff
@@ -0,0 +1,40 @@
+package kr.codesqaud.cafe.dto.comment;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import kr.codesqaud.cafe.domain.Comment;
+
+public class CommentListDto {
+    private final List<CommentReadDto> comments;
+    private final boolean hasNext;
+
+    public CommentListDto(List<CommentReadDto> comments, boolean hasNext) {
+        this.comments = comments;
+        this.hasNext = hasNext;
+    }
+
+    public List<CommentReadDto> getComments() {
+        return comments;
+    }
+
+    public boolean isHasNext() {
```

Comment:
```suggestion
    public boolean hasNext() {
```

메서드명은 이렇게 해주시고, Jackson 때문이라면 Jackson 용 애노테이션으로 처리해주세요~